docs/UserGuide.md

```diff
@@ -1,186 +1,85 @@
-
-Examples:
-
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe t/friend e/betsycrowe@gmail.com a/Newgate Prison p/1234567 t/criminal`
-
-### 2.3. Listing all persons : `list`
-
-Shows a list of all persons in the address book.<br>
+ 
+## Adding a task : `add`
+Adds a task to the task manager<br>
+Format: `add TASKNAME DATE [STARTTIME] [ENDTIME] VENUE DESCRIPTION` 
+ 
+> Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional, 
+> 
+
+Examples: 
+* `add groceries shopping 030117 0900 1200 Cold Storage Buy Extra milk`
+
+## Listing all tasks : `list`
+Shows a list of all active tasks.<br>
 Format: `list`
 
-### 2.4. Editing a person : `edit`
-
-Edits an existing person in the address book.<br>
-Format: `edit INDEX [NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
-
-> * Edits the person at the specified `INDEX`.
-    The index refers to the index number shown in the last person listing.<br>
-    The index **must be a positive integer** 1, 2, 3, ...
-> * At least one of the optional fields must be provided.
-> * Existing values will be updated to the input values.
-> * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-> * You can remove all the person's tags by typing `t/` without specifying any tags after it. 
-
-Examples:
-
-* `edit 1 p/91234567 e/johndoe@yahoo.com`<br>
-  Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@yahoo.com` respectively.
-
-* `edit 2 Betsy Crower t/`<br>
-  Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
-
-### 2.5. Finding all persons containing any keyword in their name: `find`
-
-Finds persons whose names contain any of the given keywords.<br>
+## Finding all tasks containing any keyword in their name: `find`
+Finds tasks whose names contain any of the given keywords.<br>
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
-
-Examples:
+> The search is not case sensitive, the order of the keywords does not matter, only the name is searched, 
+and tasks matching at least one keyword will be returned (i.e. `OR` search).
 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+Examples: 
+* `find groceries`<br>
+  Returns `groceries shopping` but not `groceries`
 
-### 2.6. Deleting a person : `delete`
+## Deleting a task : `delete`
+Deletes a specified tasks from the task manager. Irreversible.<br>
+Format: `delete TASKNAME`
```

Comment:
Consider: Is the TASKNAME exact match or keyword match. How are you going to do if there are multiple tasks founded. Delete them all?